models/comments.model.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection")
+const { fetchArticles } = require("./articles.models")
+
+exports.fetchCommentsByArticleId = (article_id) => {
+    let queryStr = "SELECT comments.comment_id, votes, created_at, author, body, article_id FROM comments";
+    queryStr += " WHERE comments.article_id = $1";
+    queryStr += " ORDER BY created_at DESC";
+    
+    let queryVals = []
```

Comment:
Let's declare this with a `const`, as we don't want to change this array to a different reference in memory.